build/devnet/lotus-miner/entrypoint.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -e
+echo Wait for lotus is ready ...
+lotus wait-api
+echo Lotus ready. Lets go
+if [ ! -f $LOTUS_MINER_PATH/.init.miner ]; then
+	echo Import the genesis miner key ...
+	lotus wallet import --as-default $GENESIS_PATH/pre-seal-t01000.key
```

Comment:
This step should happen in the lotus node. We are mounting /var/lib/lotus as read-only.
Not sure how this is working for other people but I get the below error.
```
2022-08-25T12:12:21.790Z	WARN	build	build/genesis.go:19	loading built-in genesis: read genesis: is a directory
ERROR: initializing node: applying node options failed: could not check lock status: cannot acquire lock: Lock Create of /var/lib/lotus/repo.lock failed: open /var/lib/lotus/repo.lock: read-only file system

```